### PR TITLE
[Fairground 🎡]  Add 5:4 aspect ratio for card pictures

### PR DESCRIPTION
**Pull request by abeddow91**
## What does this change?
Lets the consumer of CardPicture determine the aspect ratio of the image. This is limited in the types as 5:4 or 5:3 with 5:3 being the default. 

Depending on which aspect ratio is selected, the appropriate padding is added to the image. This will require editorial to select the correct crop in the grid. 

## Why?
Fronts cards are migrating to a 5:4 aspect ratio. This change allows us to introduce this aspect whilst maintaining all other card picture ratios.

## Screenshots

| Before      | After      |
| ----------- | ---------- |
| ![after][] | ![before][] |

[before]: https://github.com/user-attachments/assets/425e7648-3f97-4c8f-b7ee-c35d183fce2f
[after]: https://github.com/user-attachments/assets/df8aa7f5-5f3b-4bfc-9f3f-33803e4a0ffe

### CODE REVIEW

File: dotcom-rendering/src/components/CardPicture.tsx [hunk 1]
```diff
@@ -82,19 +84,26 @@ const block = css`
...
-		width: 100%;
-		height: 100%;
-	}
-`;
+/**
+ * On fronts, Fairground cards have an image ration of 5:4.
+ * This is due to replace the existing card ratio of 5:3
+ * For now, we are keeping both ratios.
+ */
+const decideAspectRatio = (aspectRatio: AspectRatio) => {
```
Comment (Georges-GNM) [inline dotcom-rendering/src/components/CardPicture.tsx:92]: Wonder if we can give a slightly different name for the function or prop? It feels a bit weird to see something 'deciding aspect ratio' with an aspect ratio prop. 

Maybe... eh, even something like `decideAspectRatioStyle` or something?

**Review by Georges-GNM — APPROVED** [review]
Paired on this and the logic makes sense to me 👍 

(just a small non blocking naming thing as a comment)

File: dotcom-rendering/src/components/CardPicture.tsx (hunk 1 above)
Comment (abeddow91) [reply dotcom-rendering/src/components/CardPicture.tsx:92]: good point, perhaps decideAspectRatioPadding?

Comment (abeddow91) [inline dotcom-rendering/src/components/CardPicture.tsx:92]:
```suggestion
const decideAspectRatioPaddingStyles = (aspectRatio: AspectRatio) => {
```

File: dotcom-rendering/src/components/CardPicture.tsx [hunk 2]
```diff
@@ -126,7 +136,7 @@ export const CardPicture = ({
 			data-size={imageSize}
 			css={[
 				block,
-				aspectRatio,
+				decideAspectRatio(aspectRatio),
```
Comment (abeddow91) [inline dotcom-rendering/src/components/CardPicture.tsx:139]:
```suggestion
				decideAspectRatioStyles(aspectRatio),
```

**Pull request merged**